Attempting to rename your pet throws an exception

### Check if the issue is fixed in dev builds

- [X] I have confirmed that the issue is present on the latest dev build

### What steps will reproduce the issue?

1. Have "Pets-Rename" enabled in your config.
2. Spawn a pet from the cosmetic menu.
3. Attempt to rename by clicking the rename item (or nametag).

### What was supposed to happen?

It was expected the anvil would open to be able to rename the pet.

### What happened instead?

Nothing happened (the cosmetic menu stays open) and the console threw an error in UltraCosmetics.

### What version of UltraCosmetics are you using? Do not say 'latest'.

UltraCosmetics-3.0-DEV-b4 (https://github.com/datatags/UltraCosmetics/commit/21d29f3dd084341b35dab04d58ee7cca1f78b39c)

### What server distribution (Spigot, Paper, etc.) and what version are you using? Do not say 'latest'.

This server is running Pufferfish version git-Pufferfish-54 (MC: 1.19.3) (Implementing API version 1.19.3-R0.1-SNAPSHOT) (Git: 053386a on HEAD)

### What plugins do you have installed?

Just UltraCosmetics.

### Are there any errors in the console or in logs?

```java
[19:02:49 ERROR]: Could not pass event InventoryClickEvent to UltraCosmetics v3.0-DEV-b4
java.lang.ExceptionInInitializerError: null
        at be.isach.ultracosmetics.shaded.anvilgui.anvilgui.AnvilGUI$Builder.open(AnvilGUI.java:486) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at be.isach.ultracosmetics.menu.menus.MenuPets.renamePet(MenuPets.java:82) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at be.isach.ultracosmetics.menu.menus.MenuPets.lambda$addPetRenameItem$0(MenuPets.java:54) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at be.isach.ultracosmetics.menu.Menu.onClick(Menu.java:132) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor28.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:?]        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:git-Pufferfish-54]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:674) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:3161) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:58) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:23) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1343) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1320) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1313) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1447) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1175) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318) ~[pufferfish-1.19.3.jar:git-Pufferfish-54]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalStateException: AnvilGUI does not support server version "1_19_R2"
        at be.isach.ultracosmetics.shaded.anvilgui.anvilgui.version.VersionMatcher.match(VersionMatcher.java:34) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at be.isach.ultracosmetics.shaded.anvilgui.anvilgui.AnvilGUI.<clinit>(AnvilGUI.java:35) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        ... 26 more
Caused by: java.lang.ClassNotFoundException: be.isach.ultracosmetics.shaded.anvilgui.anvilgui.version.Wrapper1_19_R2
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:179) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:126) ~[pufferfish-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:375) ~[?:?]
        at be.isach.ultracosmetics.shaded.anvilgui.anvilgui.version.VersionMatcher.match(VersionMatcher.java:28) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        at be.isach.ultracosmetics.shaded.anvilgui.anvilgui.AnvilGUI.<clinit>(AnvilGUI.java:35) ~[UltraCosmetics-3.0-DEV-b4.jar:?]
        ... 26 more
```

### Additional information that could be helpful in fixing the problem

Looks like AnvilGUI is throwing the error because it's outdated for 1.19.3?

--- a/core/build.gradle
+++ b/core/build.gradle
@@ -28,7 +28,7 @@ dependencies {
     compileOnly 'net.luckperms:api:5.4'
     compileOnly "me.lokka30:treasury-api:1.2.1-SNAPSHOT"
     implementation 'com.zaxxer:HikariCP:4.0.3'
-    implementation 'net.wesjd:anvilgui:1.5.3-SNAPSHOT'
+    implementation 'net.wesjd:anvilgui:1.6.3-SNAPSHOT'
     //implementation 'me.gamercoder215:mobchip-bukkit:1.7.1-SNAPSHOT'
     implementation(fileTree(dir: '.', include: ['MobChip-Bukkit-*.jar']))
     compileOnly 'com.discordsrv:discordsrv:1.26.0'

